db/repair.cc

```diff
@@ -544,7 +544,7 @@ class Repairer {
       InternalIterator* iter = table_cache_->NewIterator(
           ropts, file_options_, cfd->internal_comparator(), t->meta,
           nullptr /* range_del_agg */,
-          cfd->GetLatestMutableCFOptions()->prefix_extractor.get(),
+          cfd->GetLatestMutableCFOptions()->prefix_extractor,
```

Comment:
I believe this is the only place where we get the prefix_extractor from ColumnFamilyData, which is normally mutable for SetOptions but AFAIK not in Repairer.